src/main/java/org/skriptlang/skript/scheduler/AsyncTask.java

```diff
@@ -0,0 +1,185 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package org.skriptlang.skript.scheduler;
+
+import java.util.concurrent.TimeUnit;
+
+import org.bukkit.plugin.Plugin;
+
+import ch.njol.skript.Skript;
+import ch.njol.util.Closeable;
+
+/**
+ * Represents a Task that only calls it's runnable asyncronously from the main server thread.
```

Comment:
grammar: misplaced modifier, typo and possessive form
```suggestion
 * Represents a Task that calls its runnable asynchronously from the main server thread.
```